.changeset/breezy-jokes-pump.md

```diff
@@ -0,0 +1,9 @@
+---
+'@shopify/cli-hydrogen': patch
+---
+
+Clean up messaging around unlinked storefronts when running CLI commands
+
+- We will show a warning (not error) when you run a Hydrogen CLI command on a storefront that isn't linked to a storefront on Admin.
```

Comment:
Let's clarify here that it's only for Admin related links somehow? I.e. this doesn't affect dev/build commands.

Reply:
That's a good point, i'm going to update the code so it's for `env push`, `env pull`, `env list`, `deploy`. Do we not already show an error when we run `h2 dev` on unlinked storefronts?